apptests/kind/config/kind.yaml

```diff
@@ -4,13 +4,47 @@ networking:
   podSubnet: "172.16.0.0/16"
 nodes:
   - role: control-plane
+    extraMounts:
+      - hostPath: ./testdata/rook-ceph/scripts/
+        containerPath: /hack/scripts
+      - hostPath: ./testdata/rook-ceph/manifests/
+        containerPath: /hack/manifests
     kubeadmConfigPatches:
       - |
         kind: InitConfiguration
         nodeRegistration:
           kubeletExtraArgs:
             node-labels: "ingress-ready=true"
   - role: worker
+    extraMounts:
+      - hostPath: ./testdata/rook-ceph/scripts/
+        containerPath: /hack/scripts
+      - hostPath: ./testdata/rook-ceph/manifests/
+        containerPath: /hack/manifests
+    kubeadmConfigPatches:
+      - |
+        kind: InitConfiguration
+        nodeRegistration:
+          kubeletExtraArgs:
+            node-labels: "ingress-ready=true"
+  - role: worker
```

Comment:
probably not adding too much overhead, but do we have to have 2 workers?

Reply:
Yeah, still testing this. I don't think so.

Reply:
Works with one node.